app/views/agent/tasklist/ProgressSaved.scala.html

```diff
@@ -27,21 +29,18 @@
   clientDetails: ClientDetails
 )(implicit request: Request[_], messages: Messages)
 
-@signInLink = {
-  <a href="@signInUrl" class="govuk-link sign-in-link">@messages("agent.business.progress-saved.content.sign-in-link")</a>
-}
-
 @mainTemplate(title = messages("agent.business.progress-saved.title")) {
-  <div class="govuk-panel govuk-panel--confirmation">
-    <h1 class="govuk-panel__title">
-      @messages("agent.business.progress-saved.title")
-    </h1>
-    <div class="govuk-panel__body">
-      @messages("agent.business.progress-saved.content.summary", clientDetails.name, expirationDate)
-    </div>
-  </div>
-
-  <h2 class="govuk-heading-m">@messages("agent.business.progress-saved.content.subheading")</h2>
+  @govukNotificationBanner(NotificationBanner(
+    content = HtmlContent(
+      s"""
+      <h3 class="govuk-notification-banner__heading">
+        <strong>${messages("agent.business.progress-saved.content.summary", expirationDate)}</strong>
```

Comment:
```suggestion
        ${messages("agent.business.progress-saved.content.summary", expirationDate)}
```
As far as I can tell, looking at the design patten, strong tag isn't needed here